recipes-soletta/soletta/soletta_git.bb

```diff
@@ -105,6 +105,9 @@ do_configure_append() {
        sed -i -e 's/^ *FLOW_NODE_TYPE_GTK=.*/# FLOW_NODE_TYPE_GTK is not set/g' ${S}/.config
    fi
 
+   sed -i "s/^\(CONFIG_LDFLAGS=\).*/\1\"${TARGET_LDFLAGS}\"/g" ${S}/.config
```

Comment:
@barbieri I see...that would not work because here's what cml1.bbclass does:

```
cml1_do_configure() {
	set -e
	unset CFLAGS CPPFLAGS CXXFLAGS LDFLAGS
	oe_runmake oldconfig
}
```